scripts/cxg_admin.py

```diff
@@ -267,6 +267,11 @@ def cxg_remaster(ctx):
     """Cxg remaster"""
     reprocess_datafile.cxg_remaster(ctx)
 
+@cli.command()
+@click.pass_context
+def cxg_remaster_v2(ctx):
```

Comment:
what's the diff between this cmd and cxg_remaster_v2.py script?

Reply:
I'll need to consult with @ebezzi about this!

Reply:
I will run the remastering pipeline from a Jupyter notebook, so this stuff will likely not be used.